docs/docsource/reference/prometheus/index.rst

```diff
@@ -0,0 +1,146 @@
+exposed through SoundCloud's Prometheus toolkit. Exposure through the Backbeat
+API is under development.
+
+The following Prometheus metrics are available (see the _`Prometheus 
+documentation<https://prometheus.io/docs/introduction/overview/>`
+to access counter/gauge/histogram data and how to query the Prometheus API).
+Labels and their content are described below.
+
+Available Metrics
+~~~~~~~~~~~~~~~~~
+
+**zenko_replication_queued_total**
+
+  - type: counter
+  - description: number of objects queued for replication
+  - labels: origin, partition, fromLocation, fromLocationType, toLocation, 
+    toLocationType
+
+**zenko_replication_queued_bytes**
+
+  - type: counter
+  - description: number of bytes queued for replication
+  - labels: origin, partition, fromLocation, fromLocationType, toLocation, 
+    toLocationType
+
+**zenko_replication_processed_bytes**
+
+  - type: counter
+  - description: number of bytes replicated
+  - labels: origin, fromLocation, fromLocationType, toLocation, toLocationType,
+    status
+
+**zenko_replication_elapsed_seconds**
+
+  - type: histogram
+  - description: replication jobs elapsed time in seconds
+  - labels: origin, fromLocation, fromLocationType, toLocation, toLocationType,
+    status, contentLengthRange
+
+The following labels are provided to one or more of the above metrics:
+
+  - **origin:** The "origin" label identifies the service that triggered the 
+    replication, e.g. “lifecycle”. Currently, lifecycle is the  only service
+    to use replication as reflected in these metrics, but when other services 
+    under development use it, this label will be used to distinguish replication
+    tasks by the services triggering them.
+
+  - **partition:** The partition number in the “backbeat-data-mover” Kafka topic
+    where the replication task has been queued.
+
+  - **fromLocation:** The name of the source Zenko location 
+
+  - **fromLocationType:** This identifies the type of location from which the 
+    “fromLocation” call emerged: “aws_s3” for an AWS compatible location, 
+    “local” for a local storage location or a RING location, “azure” for an Azure
+    location, “gcp” for a Google Cloud location. 
+
+  - **toLocation:** The name of the target Zenko location
+
+  - **toLocationType:** This identifies the type of location to which the information
+    in “toLocation” is targeted. This follows the convention described in 
+    **fromLocation**
+
+  - **status:** This reports the status of the finished task: “success”
+    if the replication completed successfully or “error” if it did not.
+
+  - **contentLengthRange:** This range separates the metric values in different
+    content-length (object size) buckets to offer more meaningful elapsed 
+    time metrics grouped by object size ranges. The labels used are: “<10KB”,
+    “10..30KB”, 30..100KB”, […], “300GB..1TB”, “>1TB”.
+
+Kafka Metrics
+~~~~~~~~~~~~~
+
+Kafka provides its own set of Prometheus metrics, one of which is
```

Comment:
I'm thinking other Kafka metrics could also be useful, though I did not take time to go through all of them and find use cases for lifecycle monitoring. So the one I'm presenting here "kafka_consumergroup_lag" is definitely useful, there are probably others. Now I let you rephrase this appropriately :wink:

Reply:
Added a catch-all sentence at the end of the section.